winpr/libwinpr/pool/pool.c

```diff
@@ -1,3 +1,8 @@
+#ifdef __MINGW32__
```

Comment:
this looks wrong.
if `S_TP_CALLBACK_INSTANCE` and `_TP_CALLBACK_INSTANCE` are defined, then some expectation is wrong.
`_TP_CALLBACK_INSTANCE` indicates your `mingw` supports the functions and this module should not be compiled and completely disabled.

Reply:
Cf. https://github.com/FreeRDP/FreeRDP/issues/9881#issuecomment-1986952892: This wasn't an issue before https://github.com/FreeRDP/FreeRDP/pull/9423.